Patch-Source: https://github.com/starship/starship/pull/5175
--
From a1b7169bf78614555f4e10f0d6daece1dae52ccd Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Mon, 8 May 2023 18:18:23 +0200
Subject: [PATCH] build(deps): disable unnecessary/unused regex features

This will reduce the binary size by ~400 kB.
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 71eb4d0b2a6c..8a7184ef116e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -70 +70 @@
-regex = "1.7.3"
+regex = { version = "1.7.3", default-features = false, features = ["perf", "std", "unicode-perl"] }
